db/seeds.rb

```diff
@@ -5,12 +5,25 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
 Book.delete_all
+Author.delete_all
 
-50.times do
-  Book.create(
-    title: Faker::Company.catch_phrase,
-    description: Faker::Hacker.say_something_smart,
-    cover_url: Faker::Avatar.image(nil, '100x200')
+10.times do
+  author = Author.create(
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name,
+      biography: Faker::Lorem.paragraph(5)
   )
+
+  10.times do
+    Book.create(
+        title: Faker::Company.catch_phrase,
+        description: Faker::Hacker.say_something_smart,
+        cover_url: Faker::Avatar.image(nil, '100x200'),
+        author: author,
+        published_at: Faker::Time.between(DateTime.now - 100, DateTime.now)
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.